uyuni-docs-helper

```diff
@@ -34,6 +34,18 @@ GITREPO='https://github.com/uyuni-project/uyuni-docs.git'
 # Default Git reference
 GITREF='master'
 
+# use gnu-getopt on macOS
+getopt_for_macos() {
+  if [ $(uname -s) == 'Darwin' ]; then
+    if [ ! -f /usr/local/opt/gnu-getopt/bin/getopt ]; then
+      print_error
```

Comment:
```suggestion
      print_error "Please install gnu-getopt via Homebrew with: brew install gnu-getopt"
```

Reply:
The `print_error` function needs a string as an argument.

Reply:
Fixed. Thank you!